testing/snapshot.ts

```diff
@@ -171,6 +174,7 @@ async function runPrompt(
       stderr: "piped",
       args: [
         "run",
+        "-c=deno.jsonc",
```

Comment:
We cannot hardcode the config here. It could be that there is a `deno.json` or no configuration at all instead.

Reply:
Do you prefer to set it as default option or to define it at each `snapshotTest` call ?

Reply:
I test several approach and finally I set `-c=deno.jsonc` as default option like `--allow-env=SNAPSHOT_TEST_NAME`.

Reply:
I don't want to specify the config in the command, then we would need to define the config in every snapshotTest call.
But it looks like It's a bug of the upcomming workspace feature in deno: https://github.com/denoland/deno/issues/22020
So once this is fixed, the snapshotTest method should work again.

Reply:
Ok but currently doing this way overwrite the default value ([testing/snapshot.ts#L177](https://github.com/JOTSR/deno-cliffy/blob/3016d1fd4821d703352f537cc3c618bb479ce880/testing/snapshot.ts#L177)) and then break tests. Do you also want to move `--allow-env=SNAPHOT_TEST_NAME` to the broken tests or refactor if differently like 
```ts
const denoFlags = options
    ?.denoArgs
    ?.some(flag => flag.match(/^--allow-env/)) ?
    options.denoArgs : ['--allow-env=SNAPSHOT_TEST_NAME', ...(options.denoArgs ?? [])]
```

Reply:
For the config issue we don't need to do anything, we can remove the config flag from the command. The bug currently only occurs in worspace projects and once the bug is fixed it should work.
But feel free to improve the handling of the `--allow-env` flag, but we should do this in a separate PR.
I will not merge this PR until jsr is stable. As long as jsr is in open beta, I will deploy manually from this PR.